scripts/build/cross

```diff
@@ -1,8 +1,11 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
-source ./scripts/build/ldflags
+set -eu -o pipefail
 
-gox -output build/docker-{{.OS}}-{{.Arch}} \
-    -osarch="linux/arm linux/amd64 darwin/amd64 windows/amd64" \
-	--ldflags "${LDFLAGS}" \
-	github.com/docker/cli/cmd/docker
+export BUILDDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+SHELL=/bin/bash parallel ::: \
```

Comment:
Oops, `docker.Makefile` was using the wrong image!

Thanks, just pushed a fix.